.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist
```

Comment:
I looked at a similar PR in:  https://github.com/abought/demo-ember-osf/pull/16

Similar comments apply: can you please add the `coverage` folder?